src/os/appstartdelay.js

```diff
@@ -0,0 +1,38 @@
+goog.provide('os.AppStartDelay');
```

Comment:
I started creating one of these in the past, but it's not really necessary since `onSuccess` and `onFailure` are public on `goog.async.ConditionalDelay` and can be replaced directly on the instance. See [this example](https://github.com/ngageoint/opensphere/blob/master/src/os/storage/storage.js#L53).